### PR TITLE
Diagnostic completed

### PR DESCRIPTION
**Pull request by Ryaong792**
Fist of 5 would be a 3.  I thought i was a 4 but i got confused with the
questions and how to answer them.  But i did feel that i had a grasp
on the subject till today.

### CODE REVIEW

**Comment by Ryaong792** [comment]
just read the answer and I feel dumb I really over thought what was wanted for the answers.

File: lib/diagnostic.rb [hunk 1]
```diff
@@ -4,25 +4,26 @@
...
 
 # Running scripts from the command line
-#
+example1 = example.new(24)
+end.
 # In a Ruby string, write the command you would use to run a script at
 # `lib/example.rb`
 
-Diagnostic.script = 'your response here'
+Diagnostic.script = '#{@value}'
```
Comment (jordanallain) [inline lib/diagnostic.rb:12]: We were looking for

`ruby lib/example.rb`

File: lib/diagnostic.rb [hunk 2]
```diff
@@ -4,25 +4,26 @@
...
 
 # Using the REPL
 #
 # In a Ruby string, write the command you would use to enter the REPL we
 # prefer over `irb`
 
-Diagnostic.repl = 'your response here'
+Diagnostic.repl =  starwars = "star wars episode vii"
```
Comment (jordanallain) [inline lib/diagnostic.rb:19]: Close, should be: `star_wars_episode_viii`

File: lib/diagnostic.rb [hunk 3]
```diff
@@ -4,25 +4,26 @@
...
 
 # Naming conventions in Ruby
 #
 # In a Ruby string, write what you would write to properly name a variable in
 # Ruby that read in English as "star wars episode viii".
 
-Diagnostic.variable = 'your response here'
+Diagnostic.variable = '#{charactor} is really a Sith Lord'
```
Comment (jordanallain) [inline lib/diagnostic.rb:26]: Super close here, but remember we need double quotes for string interpolation in Ruby.

File: lib/diagnostic.rb [hunk 4]
```diff
@@ -38,7 +39,7 @@
 # What is the type of object in Ruby for decimal numbers? Write your answer in
 # a Ruby string.
 
-Diagnostic.decimal = 'your response here'
+Diagnostic.decimal = 'to_i'
```
Comment (jordanallain) [inline lib/diagnostic.rb:42]: We just wanted the keywords that Ruby uses which are Float and Fixnum.

File: lib/diagnostic.rb [hunk 5]
```diff
@@ -71,7 +72,8 @@
 
 # What value will be returned. Write your response as a Ruby string.
 
-Diagnostic.flow_control = 'your response here'
+Diagnostic.flow_control = 'Batman is the darknight and bruce wayne is just your average
```
Comment (jordanallain) [inline lib/diagnostic.rb:75]: Should be `The Dark Knight`

File: lib/diagnostic.rb [hunk 6]
```diff
@@ -82,7 +84,7 @@
 # Does ruby require an explicit return from methods? Answer true or false, and
 # replace nil.
 
-Diagnostic.returns = nil
+Diagnostic.returns = true
```
Comment (jordanallain) [inline lib/diagnostic.rb:87]: Actually it is `False`

**Comment by jordanallain** [comment]
0/15

Hey Ryan looks like you struggled with this but I bet you just misunderstood some of the directions. Let me know if you want to go over any of these with me.

**Pull request closed without merging**